CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 0.3.0
+
+- Signature changes in configuration.
```

Comment:
Pankaj, can you give some more details around the signature change, so that existing developers are aware of the changes.

Reply:
we are converting class base configuration to dictionary base with key and values . I this info we can give .